downstream/modules/platform/con-controller-control-nodes.adoc

```diff
@@ -0,0 +1,10 @@
+[id="con-controller-control-nodes"]
+
+= Control nodes
+
+Vertically scaling a control node increases the number of jobs it can perform control tasks for, which requires both more CPU and memory. 
+In general, scaling CPU alongside memory in the same proportion is recommended (for example, 1 CPU: 4GB RAM). 
+Even in the case where memory consumption is observed to be high, increasing the CPU capacity of an instance can often relieve pressure, as most memory consumption of control nodes is usually from unprocessed events.
+
+As mentioned in the xref:ref-controller-performance-improvements[Performance improvements], increasing the number of processors can also increase the job event processing rate of a control node. 
+At this time, vertically scaling a control node does not increase the number of workers that handle web requests, so horizontally scaling is more effective, if the desire is to increase the API availability.
```

Comment:
Suggest rewording to: 
At this time, vertically scaling a control node does not increase the number of workers that handle web requests, so scaling horizontally is more effective for increasing the API availability.